### PR TITLE
Add dev dependencies to package.json

### PR DESCRIPTION
**Pull request by chrisdwheatley**
Just a small one, I was trying to get the app up and running and noticed there were no dependencies to install, this should fix that. I also updated to readme to use the local version of gulp for those who may not have it available globally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,9 @@ Angular 2 is not packaged or ready for production... but this is a todo app with
 
 ### Installation
 
-```bash
-npm install
-gulp
-gulp serve
-```
+`npm install`
```
Comment (davideast) [inline README.md]: Can you break this out into one nice to read code block?

Something like:

``` bash
npm install -g gulp # Skip if you have gulp
npm install
gulp
gulp serve
```

**Comment by davideast** [comment]
Thanks for the PR! I just have one minor comment.

Thanks for bringing this to my attention. The installation setup will not work without this.

**Comment by chrisdwheatley** [comment]
Sure thing.

**Comment by chrisdwheatley** [comment]
Updated.

**Comment by davideast** [comment]
Thanks!

**Pull request merged**